src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java

```diff
@@ -81,6 +81,24 @@ public class HydrateBreakMessageDictionary {
                     }}
             );
     
+    private static final List<String> HYDRATE_BREAK_ROMANTIC_TEXT_LIST =
+            Collections.unmodifiableList(
+                new ArrayList<String>(){{
```

Comment:
Can you add a space between the `()` and `{{` here?

Reply:
Yeah, I've added it.